pkg/apis/controller/experiments/v1beta1/experiment_types.go

```diff
@@ -187,8 +188,36 @@ type FeasibleSpace struct {
 }
 
 type TrialTemplate struct {
-	Retain     bool        `json:"retain,omitempty"`
+	// Retain indicates that Trial resources must be not cleanup
+	Retain bool `json:"retain,omitempty"`
+
 	GoTemplate *GoTemplate `json:"goTemplate,omitempty"`
```

Comment:
Are we supporting Gotemplate also?

Reply:
All current examples in v1beta1 are working with `GoTemplate`.
You think we should remove it and don't run e2e test until we implement new logic?

Reply:
I think we should remove and disable test if we decide to remove it.

Reply:
I agree.  Lets make all changes to trial template API in a single PR for better tracking . Once new TrialTemplate implementation is added, move all examples to new trial template.

Reply:
Ok, I will remove `GoTemplate` in this PR and disable all e2e tests.